src/obfl-specification.html

```diff
@@ -1606,6 +1606,62 @@ <h4 id="L1105" class="include">toc-entry</h4>
 </dl>
 </div>
 
+<h4 id="L1105" class="include">toc-entry-on-resumed</h4>
+
+<p>The toc-entry-on-resumed element defines an entry in a table of contents. It is a
+conditional container of content for volume breaks. The contents of at most one
+toc-entry-on-resumed per volume will be rendered at the start of the resumed volume's
+table of contents.
```

Comment:
Thought I already commented on this but somehow got lost:

Technically there is no reason why we couldn't render more than one toc-entry-on-resumed per volume. Apart from this, the sentence is also just confusing. If you read it carefully it says a lot of different things, and makes a few too many assumptions. Can we just drop it?

Reply:
Suppose you have a `toc-block` with `<toc-entry-on-resumed range=`_range1_`>`Continuation of Chapter 1</toc-entry-on-resumed>`, and it has a child `toc-block` with `<toc-entry-on-resumed range=`_range1.1_`>Continuation of Chapter 1.1</toc-entry-on-resumed>`. If the volume break occurs in _range1.1_, the `Continuation of Chapter 1` should **NOT** be shown.

Reply:
We don't want that kind of behavior. If you don't want to show a `toc-entry-on-resumed` is a certain range you have to fix the `range` attribute. If the range is not continuous you need to create several `toc-entry-on-resumed` for the different parts of the range. I posted some examples of this before. If needed we can extend the `range` attribute to allow multiple parts (similar to Maven).

Reply:
Ok, that is not how I understood it, but it makes things simpler.